wordpress-mobile/gutenberg-mobile
# Update GB reference to fix web breakage in Clipboard.

Fixes a web breakage related to use of clipboard being converted to a hook that is imported by native code now.

To test:
 - Build and run the demo app
 - Smoke test it to see if all is good.

PR submission checklist:
